INSTALL.md

```diff
@@ -38,10 +38,22 @@ Dangerzone is available for:
   ```
 </details>
 
-Add our repository following [these instructions](https://packagecloud.io/firstlookmedia/code/install#manual-deb), or by running this script:
-
-```
-curl -s https://packagecloud.io/install/repositories/firstlookmedia/code/script.deb.sh | sudo bash
+Add our repository following these instructions:
+
+```sh
+sudo apt-get update
+sudo apt-get install -y curl
+sudo mkdir -p /etc/apt/keyrings/
+curl -S https://packages.freedom.press/apt-tools-prod/fpf-archive-keyring.gpg \
+    | sudo tee >/dev/null /etc/apt/keyrings/fpf.gpg
+source /etc/os-release
+echo deb [signed-by=/etc/apt/keyrings/fpf.gpg] https://packages.freedom.press/apt-tools-prod ${VERSION_CODENAME?} main \
```

Comment:
`${VERSION_CODENAME?}` since all debian versions are in fact the same, is it possible for us to do [like the signal folks](https://signal.org/download/#) and just hardcode here one version and avoid an extra line for the user to run? (the `source` line).

Reply:
We discussed this live. This may become an issue if we want to ship different package definitions for other repos (e.g., compiled code, extra dependencies, etc.). I suggest we start conservatively for now.